instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/package.py

```diff
@@ -15,3 +15,5 @@
 _instrumenting_module_name = "opentelemetry.instrumentation.sqlalchemy"
```

Comment:
This is probably not needed anymore right?

Reply:
Right! I will remove this